runelite-client/src/main/java/net/runelite/client/plugins/lmsperformancetracker/TotalStatsPanel.java

```diff
@@ -0,0 +1,131 @@
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.lmsperformancetracker;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.GridLayout;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
+import javax.swing.border.EmptyBorder;
+import net.runelite.client.ui.ColorScheme;
+
+// basic panel with 3 rows to show a title, total fight performance stats, and kills/deaths
+public class TotalStatsPanel extends JPanel
+{
+	private JLabel statsLabel;
+	private JLabel killsLabel;
+	private JLabel deathsLabel;
+	private FightPerformance totalStats;
+	private int numKills = 0;
+	private int numDeaths = 0;
+
+	TotalStatsPanel()
+	{
+		totalStats = new FightPerformance("Player", "");
+
+		setLayout(new GridLayout(3, 1));
+		setBorder(new EmptyBorder(8, 8, 8, 8));
+		setBackground(ColorScheme.DARKER_GRAY_COLOR);
+
+		// basic label to display a title.
+		JLabel titleLabel = new JLabel();
+		titleLabel.setText("LMS Performance Tracker");
+		titleLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		titleLabel.setForeground(Color.WHITE);
+		add(titleLabel);
+
+		// panel to show the total fight stats (successful hits/total attacks)
+		JPanel statsPanel = new JPanel(new BorderLayout());
+
+		JLabel leftLabel = new JLabel();
+		leftLabel.setText("Total Stats:");
+		leftLabel.setForeground(Color.WHITE);
+		statsPanel.add(leftLabel, BorderLayout.WEST);
+
+		statsLabel = new JLabel();
+		statsLabel.setToolTipText(totalStats.getPlayerSuccessCount() + " successful off-pray attacks/" +
+			totalStats.getPlayerAttackCount() + " total attacks");
+		statsLabel.setText(totalStats.getPlayerStatsString());
+		statsLabel.setForeground(Color.WHITE);
+		statsPanel.add(statsLabel, BorderLayout.EAST);
+		statsPanel.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+
+		add(statsPanel);
+
+		// panel to show total kills/deaths
+		JPanel killDeathPanel = new JPanel(new BorderLayout());
+
+		killsLabel = new JLabel();
+		killsLabel.setText(numKills + " Kills");
+		killsLabel.setForeground(Color.WHITE);
+		killDeathPanel.add(killsLabel, BorderLayout.WEST);
+
+		deathsLabel = new JLabel();
+		deathsLabel.setText(numDeaths + " Deaths");
+		deathsLabel.setForeground(Color.WHITE);
+		killDeathPanel.add(deathsLabel, BorderLayout.EAST);
+
+		killDeathPanel.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+
+		add(killDeathPanel);
+	}
+
+	public void addFight(FightPerformance fight)
+	{
+		if (fight.isPlayerDied())
```

Comment:
It's think it's better to just use one variable to track if the player won or lost the fight. Then you can just do `if/else`